c2corg_api/models/association.py

```diff
@@ -212,6 +212,8 @@ def get_linked_routes(document):
 
     return limit_route_fields(
         DBSession.query(Route).
+        options(joinedload(Route.geometry).load_only(
+            DocumentGeometry.geom_detail)).
```

Comment:
You also have to change it here:
https://github.com/c2corg/v6_api/blob/master/c2corg_api/views/waypoint.py#L316